### DISCUSSION TITLE
node.setErrors but for Success Messages

### DISCUSSION THREAD

**Discussion by pano9000** [formkit/formkit#993] [Ideas]
Hi All,

quick idea, because I did not  find anything similar in the FormKit docs (or am I missing something obvious here?):

I feel like it would be useful to also have a method similar to the `node.setErrors` one, but one that is used to set "Success Messages", received from the backend (or some other source), e.g. "The Form has been submitted successfuly."

This could then be used together with FormKitMessages, to display that message, after the Form was successfully submitted.

It feels wrong to use node.setErrors for this currently.

**Comment by fenilli** [comment]
You can use the message store system for that: https://formkit.com/essentials/architecture#message-store

**Reply by pano9000** [reply]
thank you, I will take a look at this and report back.

**Comment by justin-schroeder** [comment]
@pano9000 you’re right though that success messages as a first-class feature is something we should explore more.

**Comment by pano9000** [comment]
so I just got to spend some time with it and while it definitely can be used to achieve what I want, it requires a bit of extra work:
The messages by default are still shown with red text color (which indicates an error), so I'd need to now do some extra work to make that text look less like an error and so on.

So I feel like something like `node.setSuccess` (which would set some sensible styling defaults) to easily handle that could be a good idea for the future.

@justin-schroeder thank you for acknowledging the idea! :-)
